src/Components/Input/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { View, Text, TextInput } from 'react-native';
+
+import { styles } from './styles';
+import { IInput } from './types';
+
+const Input: React.FC<IInput> = ({ placeholder, fontSize }) => (
+  <View style={styles.container}>
+    <TextInput style={{ fontSize: fontSize }} placeholder={placeholder} />
```

Comment:
Considerando o comentário acima, acredito que aqui a intenção era passar o `style.input`:
```suggestion
    <TextInput style={styles.input} placeholder={placeholder} />
```